planner/core/explain.go

```diff
@@ -364,12 +364,14 @@ func (p *PhysicalSort) ExplainInfo() string {
 
 // ExplainInfo implements Plan interface.
 func (p *PhysicalLimit) ExplainInfo() string {
-	var str strings.Builder
-	str.WriteString("offset:")
-	str.WriteString(strconv.FormatUint(p.Offset, 10))
-	str.WriteString(", count:")
-	str.WriteString(strconv.FormatUint(p.Count, 10))
-	return str.String()
+	buffer := bytes.NewBufferString("")
```

Comment:
I think it is better to differentiate Limit with partition by calling it WindowLimit and do not include partition by fields.

Reply:
You mean adding a new `PhysicalWindowLimit` or just display as `WindowLimit` in the explain info? Adding a new `PhysicalWindowLimit` need a lots of changes, and I think we will miss the deadline if we wants to this :(